sub.go

```diff
@@ -29,6 +29,7 @@ type SubParams struct {
 	Items                                           []*SubItemsParams
 	Billing                                         SubBilling
 	DaysUntilDue                                    uint64
+	OnBehalfOf                                      string
```

Comment:
@jzs This is undocumented, the `OnBehalfOf` is only supported on Charge creation.

Reply:
@remi-stripe @brandur-stripe The server side api does actually support `on_behalf_of` when creating subscriptions and we do need that feature to keep charges in correct currency.
However if there are reasons it can't be included we would just have to fork the client which is always not the preferred option :)

Reply:
I'm never a big fan of merging things that are not really supported but if you do need it I don't see a safer/better solution in this case.